docs/pages/versions/unversioned/sdk/filesystem.mdx

```diff
@@ -584,7 +584,9 @@ Returns an object with the following fields:
 
 ### `FileSystem.getContentUriAsync(fileUri)`
```

Comment:
Hi @StCyr,

These docs have been moved to more "autogenerating" API recently. You will have to add a `platform @Android` tag in this file (the content of this function starts from line number 92): https://github.com/expo/expo/blob/3329e04d849b3ce0de9f59d38d66a29455573c80/packages/expo-file-system/src/FileSystem.ts#L92

This will indicate that this function is to be used only for Android as shown below:

<img width="290" alt="CleanShot 2022-11-28 at 00 06 03@2x" src="https://user-images.githubusercontent.com/10234615/204153382-fe215e4c-804e-40a0-a22e-148228fec1b7.png">

After you make the changes, I can backport them to the SDK versions. 

Sorry for the long process 😅